### PR TITLE
docker/docker-compose config for sync server

### PR DESCRIPTION
**Pull request by jjidt**

### CODE REVIEW

File: sync/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+FROM node:12.13.0-alpine
+
+# bcrypt and alpine linux have some issues:
+# https://github.com/kelektiv/node.bcrypt.js/wiki/Installation-Instructions#alpine-linux-based-images
+RUN apk --no-cache add --virtual builds-deps build-base python
```
Comment (dheniges) [inline sync/Dockerfile]: I don't think you need these lines?

File: sync/Dockerfile-prod [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+FROM node:12.13.0-alpine
+
+# bcrypt and alpine linux have some issues:
+# https://github.com/kelektiv/node.bcrypt.js/wiki/Installation-Instructions#alpine-linux-based-images
+RUN apk --no-cache add --virtual builds-deps build-base python
```
Comment (dheniges) [inline sync/Dockerfile-prod]: Same. I don't see bcrypt in sync which is why I had to add these dependencies

**Review by dheniges — CHANGES_REQUESTED** [review]
Couple things can be removed from the docker images -- see comments. Otherwise looks good

**Comment by jjidt** [comment]
updated

**Review by dheniges — APPROVED** [review]
still builds 👍

**Pull request merged**